client/sysinfo/src/sysinfo.rs

```diff
@@ -319,6 +334,41 @@ pub fn benchmark_disk_random_writes(directory: &Path) -> Result<u64, String> {
 
 	// We only wrote half of the bytes hence `SIZE / 2`.
 	benchmark("disk random write score", SIZE / 2, MAX_ITERATIONS, MAX_DURATION, run)
+		.map(|s| s as u64)
+}
+
+pub fn benchmark_sr25519_verify(limit: ExecutionLimit) -> f64 {
+	const INPUT_SIZE: usize = 32;
+	const ITERATION_SIZE: usize = 2048;
```

Comment:
shouldn't these event be configurable?

Reply:
I will extend the whole configuration of the `sc-sysinfo` a bit in the future, but for these two its fine.  

`INPUT_SIZE`: as @koute pointed out, having this as variable makes the results in-comparable to each other, as each input size is basically another benchmark.  
`ITERATION_SIZE` itself does not change the result of the benchmark. It is only to ensure that the time deadline can be met with coarse precision.